libsrc/c64/ser/c64-up2400.s

```diff
@@ -0,0 +1,476 @@
+; Serial driver for the C64 using Userport.
+;
+; Johan Van den Brande, (c) 2014
+; 
+; Based on George Hug's 'Towards 2400' article in the Transactor Magazine volume 9 issue 3.
+; (https://archive.org/details/transactor-magazines-v9-i03) 
+;
+        .include        "cbm_kernal.inc"
+        .include        "zeropage.inc"
+        .include        "ser-kernel.inc"
+        .include        "ser-error.inc"
+        .include        "c64.inc"
+
+        .macpack        module
+
+; ------------------------------------------------------------------------
+; Constants
+ribuf = $f7
+robuf = $f9
+baudof = $0299
+ridbe = $029b
+ridbs = $029c
+rodbs = $029d
+rodbe = $029e
+enabl = $02a1
+rstkey = $fe56
+norest = $fe72
+return = $febc
+oldout = $f1ca
+oldchk = $f21b
+findfn = $f30f
+devnum = $f31f
+nofile = $f701
+
+; ------------------------------------------------------------------------
+; Header. Includes jump table
+
+        module_header   _c64_up2400_ser
```

Comment:
No other serial driver has the max baud speed in its name. I vote for calling it `_c64_userport_ser` or `_c64_up_ser`.

Reply:
it makes sense - to seperate it from a driver for daniel dallmanns 9600 baud hack (which requires some extra connections), which is commonly referred to as "up9600"

perhaps drivers that dont require extra hardware should generally have "std" in their names?

Reply:
Kinda off-topic: where can I get an userport interface with or w/o "daniel dallmann's hack for 9600"?

Reply:
if you arent afraid of some (very easy) soldering, you can make it yourself. easiest is to get a TTL->rs232 dongle from ebay and just add a userport connector. i recommend something like this: https://gglabs.us/node/530

Reply:
> i recommend something like this: https://gglabs.us/node/530

I think I could build that from the schematics. But browsing their site I found that: https://gglabs.us/node/2044
Shouldn't that be the same? (When I have the choice between building myself or buying a prebuilt part, I typically choose the latter. :-) )

Reply:
And, another, sorry for the now really off-topic, question: What kind of adapter would I need to try the Plus/4 implemetation on hardware?

Reply:
@groessler http://cbm-hackers.2304266.n4.nabble.com/userport-C64-Plus4-others-tp4447868p4450398.html